keyboards/jakob33/readme.md

```diff
@@ -0,0 +1,27 @@
+# jakob33
+
+![jakob33](imgur.com image replace me!)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Lalit Mistry](https://github.com/schwarzer-geiger)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```

Comment:
Please fill this out.

Reply:
Thanks, I don't want to bring any of this yet to the official qmk repo.

Reply:
So at this point my PR is only in relation to the edinburgh41 folder. Can you handle that? I know this is not an ideal way of doing it and I should instead commit only the relevant folder. I am currently trying to figure out how to do that, I'm learning.